apps/backend/tsconfig.json

```diff
@@ -5,5 +5,5 @@
     "baseUrl": "./",
     "allowJs": true
   },
-  "exclude": ["node_modules", "dist", "**/*.spec.ts", "./env.js"]
+  "exclude": ["node_modules", "**/*.spec.ts", "./env.js", "./dist"]
```

Comment:
I'm not sure if this would be relevant for this PR, but can we use absolute imports? My autoimport always imports 'src/ ...etc' which doesn't show up as an error until I run the lambda handler locally